_includes/shared/code/ccloud/config.properties

```diff
@@ -0,0 +1,15 @@
+# Required connection configs for Kafka producer, consumer, and admin
+bootstrap.servers=pkc-4kgmg.us-west-2.aws.confluent.cloud:9092
```

Comment:
maybe replace `pkc-4kgmg.us-west-2.aws.confluent.cloud:9092` with {{ BOOTSTRAP SERVER }}

Reply:
yes 💯 I addressed this in https://github.com/confluentinc/kafka-tutorials/pull/786/commits/4fe0819911b66287416c45d795eadb35ab3930a7